releasenotes/notes/fix-qpy-symengine-compat-858970a9a1d6bc14.yaml

```
@@ -0,0 +1,62 @@
+---
+fixes:
+  - |
+    Fixed an issue with :func:`.qpy.load` when loading a QPY file containing
+    a :class:`.ParameterExpression`, if the versions of ``symengine`` installed
+    in the generating and loading environments were not the same.  For example,
+    if a QPY file containing :class:`.ParameterExpression`\ s was generated
```

We discourage using the style `MyObject`s because it reads poorly and does not translate well. Instead, you could use

```suggestion
    if a QPY file containing :class:`.ParameterExpression` objects was generated
```

> and does not translate well.

(Still TBD if we're going to globalize API references)